tests/shared/src/test/scala/matryoshka/spec.scala

```diff
@@ -1018,27 +1018,29 @@ class MatryoshkaSpecs extends Specification with ScalaCheck with ScalazMatchers
   }
 
   "Attr" >> {
+    type T[A] = Cofree[Exp, A]
+
     "attrSelf" >> {
       "annotate all" >> Prop.forAll(expGen) { exp =>
-        exp.cata(attrSelf[Mu[Exp], Exp]).elgotPara(universe) must
-          equal(exp.elgotPara(universe).map(_.cata(attrSelf[Mu[Exp], Exp])))
+        exp.cata(attrSelf[T[Mu[Exp]]].apply).elgotPara(universe) must
```

Comment:
Aye, but the inner `apply` in `attrSelf` takes no arguments, so if you ever want to invoke it directly you'll have to use `apply` or completely fill in both type parameter lists 😞. Anyhow, I think the consistency is worth, so /shrug.